### PR TITLE
Fix typo which is causing install failures. Bump patch version.

### PR DESCRIPTION
**Pull request by dsully**
## Description
Fix typo which is causing install failures in our build pipeline. Bump patch version.

## Types of Changes

- [ ] Core
- [X] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)

**Review by patrick91 — APPROVED** [review]
thanks!

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "strawberry-sqlalchemy-mapper"
 packages = [ { include = "strawberry_sqlalchemy_mapper", from = "src" } ]
-version = "0.2.0"
+version = "0.2.1"
```
Comment (patrick91) [inline pyproject.toml:4]:
```suggestion
version = "0.2.0"
```

this is updated automatically :)

**Comment by dsully** [comment]
Thanks - once this is merged, would you be able to get a pypi release out?

**Comment by patrick91** [comment]
@dsully yes, that's all automated! I'll merge it now!

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Comment by patrick91** [comment]
@dsully it should be out now, let me know if there's any other issue :)

**Pull request merged**